src/pages/registration/components/RegistrationNotFound.tsx

```diff
@@ -0,0 +1,39 @@
+import { Container, Heading, Flex, Text, Divider } from '@chakra-ui/layout';
+import { Button, Box } from '@chakra-ui/react';
+import { HiExternalLink } from 'react-icons/hi';
+import { useParams } from 'react-router';
+
+import { useDarkMode } from 'lib/hooks/useDarkMode';
+import { getReadableTerm } from 'lib/utils/terms';
+
+export function RegistrationNotFound() {
+  const { term } = useParams();
+  const mode = useDarkMode();
+  return (
+    <Container maxW="container.xl" centerContent data-testid="registration-component-1">
```

Comment:
```suggestion
    <Container maxW="container.xl" centerContent data-testid="courses-not-found">
```
or something is more descriptive